### PR TITLE
Disabled button style

### PR DESCRIPTION
**Pull request by mic-web**
Resolves: https://github.com/mll-lab/react-components/issues/106

Example screenshots of resulting styles:

`<Button>`

image

`<Button disabled>`

image

`<InfoButton>`

image

`<InfoButton disabled>`

image

Also fixes the `InfoButton`, which was not visible in the stories before and which had a very low contrast (on a bright background). @spawnia @SimBig is this change ok, or was the very light gray necessary?

### CODE REVIEW

**Review by spawnia — APPROVED** [review]

File: src/theme.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const THEME = {
   successColor: PALETTE.green,
   warningColor: PALETTE.gold,
   errorColor: PALETTE.red,
-  infoColor: PALETTE.gray4,
+  infoColor: PALETTE.gray7,
```
Comment (SimBig) [inline src/theme.ts:54]: @mic-web is it safe to change this? With this change, we change in all colors in which infocolor is referenced in limes.

**Comment by mic-web** [comment]
Ich bin vom 17.12.2021  bis 30.12.2021  nicht erreichbar. Ihre E-Mail wird nicht weitergeleitet. In dringenden Fällen wenden Sie sich bitte an Herrn Igor Podolskiy ***@***.***  ***@***.***, Tel. +49-89-99017503).

Mit freundlichen Grüßen
Michael Adlfinger

I am out of office from 17.12.2021  till 30.12.2021 . Your email will not be forwarded. For urgent matters please contact Mr. Igor Podolskiy ***@***.***, phone +49-89-99017503).
Best regards
Michael Adlfinger
@SimBig commented on this pull request.



>  export const THEME = {
   successColor: PALETTE.green,
   warningColor: PALETTE.gold,
   errorColor: PALETTE.red,
-  infoColor: PALETTE.gray4,
+  infoColor: PALETTE.gray7,

@mic-web is it safe to change this? With this change, we change in all colors in which infocolor is referenced in limes.

-- 
Reply to this email directly or view it on GitHub:
https://github.com/mll-lab/react-components/pull/111#pullrequestreview-836224585
You are receiving this because you were mentioned.

Message ID: ***@***.***>

To: "mll-lab/react-components" ***@***.***>

Cc: "mic-web" ***@***.***>

    "Mention" ***@***.***>

Reply-To: "mll-lab/react-components" ***@***.***>

File: src/theme.ts (hunk 1 above)
Comment (mic-web) [reply src/theme.ts:54]: In limes it doesn't make a change, as the `infoColor` is overwritten with the limes-frontend `theme`, passed to the `Provider`. It only changes this component library's default color.

**Review by SimBig — APPROVED** [review]

**Comment by mic-web** [comment]
I changed the text color of disabled buttons a bit, as previously it was not obvious enough that the button is disabled.

There's a bit less contrast now, which aligns more with the default antd design. By default, the button use the `@disabled-color` defined within the `antd.less` for its text color. As we darkened this color to improve readability of disabled elements, I think this should not apply for buttons.

**Pull request merged**